multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -43,12 +43,15 @@ coreos:
         Environment=KUBELET_ACI={{.HyperkubeImageRepo}}
         Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf \
         --mount volume=dns,target=/etc/resolv.conf \
-        --volume=rkt,kind=host,source=/opt/bin/host-rkt \
+        --volume rkt,kind=host,source=/opt/bin/host-rkt \
         --mount volume=rkt,target=/usr/bin/rkt \
         --volume var-lib-rkt,kind=host,source=/var/lib/rkt \
         --mount volume=var-lib-rkt,target=/var/lib/rkt \
-        --volume=stage,kind=host,source=/tmp \
-        --mount volume=stage,target=/tmp"
+        --volume stage,kind=host,source=/tmp \
```

Comment:
@colhom 

> What is the motivation for host mounting `/tmp`?
>
> Our setup actually mounts in an entirely separate directory to store cursor positions for our aggregrator (currently fluentd) across pod lifecycle. I suspect maybe that's what this is for?

The `/mnt` mounting already existed there; I only removed the `=` there after `--volume` (to keep it consistent with the other flags).